probe/kubernetes/kubelet_test.go

```diff
@@ -0,0 +1,73 @@
+package kubernetes_test
```

Comment:
I was hoping I would do this here, but making those symbols private imply making `reporter_rest.go` not import the kubernetes package, which implies a rather large refactoring.